parent/pom.xml

```diff
@@ -140,7 +140,6 @@
         <zkclient.version>0.5</zkclient.version>
```

Comment:
Should be removed:
<swagger2.version>2.9.2</swagger2.version>
<commons.compress.version>1.18</commons.compress.version>
<commons.lang3.version>3.9</commons.lang3.version>

Reply:
Fixed